integration-tests/src/test/groovy/com/okta/sdk/tests/it/AuthorizationServerIT.groovy

```diff
@@ -43,10 +55,463 @@ class AuthorizationServerIT extends ITSupport {
+
+        Policy policy = client.instantiate(Policy)
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition).setInclude(["ALL_CLIENTS"])))
+        registerForCleanup(policy)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        Policy createdPolicy = createdAuthorizationServer.createPolicy(policy)
+        registerForCleanup(createdPolicy)
+
+        assertThat(createdPolicy, notNullValue())
+
+        createdPolicy.setName("Test Policy Updated")
+        createdPolicy.setDescription("Test Policy Updated")
+
+        Policy updatedPolicy = createdAuthorizationServer.updatePolicy(createdPolicy.getId(), createdPolicy)
+        assertThat(updatedPolicy, notNullValue())
+        assertThat(updatedPolicy.getId(), equalTo(createdPolicy.getId()))
+        assertThat(updatedPolicy.getName(), equalTo(createdPolicy.getName()))
+        assertThat(updatedPolicy.getDescription(), equalTo(createdPolicy.getDescription()))
+    }
+
+    @Test
+    void deleteAuthorizationServerPolicyTest() {
+        String name = "java-sdk-authorization-server-" + RandomStringUtils.randomAlphanumeric(15)
+
+        Policy policy = client.instantiate(Policy)
+            .setType(PolicyType.OAUTH_AUTHORIZATION_POLICY)
+            .setName("Test Policy")
+            .setDescription("Test Policy")
+            .setPriority(1)
+            .setConditions(client.instantiate(PolicyRuleConditions)
+                .setClients(client.instantiate(ClientPolicyCondition).setInclude(["ALL_CLIENTS"])))
+        registerForCleanup(policy)
+
+        AuthorizationServer createdAuthorizationServer = client.createAuthorizationServer(
+            client.instantiate(AuthorizationServer)
+                .setName(name)
+                .setDescription("Test Authorization Server")
+                .setAudiences(["api://default"]))
+        registerForCleanup(createdAuthorizationServer)
+
+        assertThat(createdAuthorizationServer, notNullValue())
+
+        Policy createdPolicy = createdAuthorizationServer.createPolicy(policy)
+        registerForCleanup(createdPolicy)
+
+        assertThat(createdPolicy, notNullValue())
+
+        createdAuthorizationServer.deletePolicy(createdPolicy.getId())
+
+        sleep(2000)
+
+        try {
+            client.getPolicy(createdPolicy.getId())
```

Comment:
see above `expectException` commnet

Reply:
Noted.

Reply:
🤔 we probably _don't_ need to check to make sure the server actually deleted the resource.
As that is outside the scope of what the SDK has control over.

I'm not opposed to keeping it this way, we do this in a few places, but removing this pattern would speed up the ITs (at the cost of validating the 404 response)

Reply:
Agreed on the overhead of triggering a request and validating a expected 404.

Reply:
Updates done.